### ISSUE TITLE
Improper use of `aria-hidden`

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-modules/FacetedBrowse#68]
`#section-content` uses `aria-hidden="true"`. This is being blocked by browsers as focusable elements are nested under this div, but I don't see a reason for its use here as it does otherwise inherit to its descendants, which should all be receiving focus. This could have consequences for assistive tech if it goes unblocked.

**Comment by erinchambers** [comment]
Was just headed here to report the same. As a result of the `aria-hidden="true"`, the faceted browse results and pagination absent from the accessibility tree. Screen readers cannot access the search results; they go straight from the facets to the footer.

Tested using Voiceover screen reader on macOS Sequoia 15.6.1., and the accessibility tree in Firefox 142.0.1 and Safari 18.6.

**Comment by kimisgold** [comment]
This seems to be an issue coming from the javascript that manages the facet modal that's used on mobile device widths. I've pushed a fix that's ready for a look on the `aria-hidden-fix` branch. @aviannamiller could you try it out?

**Comment by aviannamiller** [comment]
Things now look good in standard desktop viewports; however, there are some issues in smaller viewports:
- When a viewport shrinks to ≤ 895px wide, `aria-hidden` sets to true, even before the facets move behind the "Filters" button into a modal (and thus, before that modal is even opened). In order for it change back to false (besides widening the viewport), the "Filters" modal must be opened, then closed. Then, `aria-hidden` sets to false.
- When the modal is open, focus can move behind it to elements that should be non-focusable.

This is true for Chrome, Safari, and Firefox, desktop and mobile (viewports ≤ 895px).

**Comment by kimisgold** [comment]
@erinchambers Thanks for bringing this back to our attention. I've resumed work on a fix for this.

@aviannamiller I have a new branch `dialog-fix` that looks to avoid all this `aria-hidden` juggling by using `<dialog>` to manage focus trapping on mobile views. Can you take a look and see if it addresses the initial issues?

**Comment by aviannamiller** [comment]
Things are much improved in terms of focus remaining in the proper regions and in the expected order.

* I tested with a long list of filters and noticed that the dialog does not appear to be scrollable, so the filters are cut off. This is true for all tested browsers (Safari, Firefox, Chrome). 
* Between viewport widths of 640 - 895 px, the sidebar disappears, and the “Filters” button is not yet visible, so there is no visible way to interact with the filters.
* My only other feedback is with the styling of the buttons that trigger the dialog or are otherwise present within it. The following could also shift to a separate issue if that feels more appropriate.
    * For Center Row, text to bg contrast is only 4.2:1 for “Filters” and “Back” button within dialog
    * Cozy, dialog close button is 1.2:1, 1:1 on hover
    * Freedom, the “Back” button within the dialog could use some adjustments to size, positioning, etc.:
       <img width="492" height="541" alt="Detail of Filters dialog with short list of available filters. Oversized button is positioned at the end of the last bit of text in the list and appears to be taking up the equivalence of about five lines of text in height" src="https://github.com/user-attachments/assets/62a387f7-2be4-4969-b8bc-9cf326bb93d4" />
    * For Foundation, the "Filters" button is pretty indistinguishable:
      <img width="596" height="161" alt="'Filters' button positioned above list of resources. Appears as plain text." src="https://github.com/user-attachments/assets/9923ecd3-b00f-4d30-aca6-1e193ad9836a" />
    * The “Filters” button on other themes not yet mentioned here could generally use some adjusted padding, etc.

**Comment by kimisgold** [comment]
I think I've addressed the main issues addressed here, ready for another look.

**Comment by aviannamiller** [comment]
Changes look great, thanks! This is all set to close.

**Comment by kimisgold** [comment]
Fixed in https://github.com/omeka-s-modules/FacetedBrowse/pull/74 and merged.
